src/CLIOptions.cpp

```diff
@@ -103,17 +132,22 @@ void setCLIOptions(CLI::App &App, MainOpt &MainOptions) {
       ->required();
   addUriOption(App, "--status-uri", MainOptions.KafkaStatusURI,
                MainOptions.ReportStatus,
-               "<host[:port][/topic]> Kafka broker/topic to publish status "
+               "<//host[:port][/topic]> Kafka broker/topic to publish status "
```

Comment:
```suggestion
               "<host[:port][/topic]> Kafka broker/topic to publish status "
```
To avoid reverting recent changes.